tests/python/relax/test_transform_legalize_ops_manipulate.py

```diff
@@ -18,7 +18,7 @@
 import pytest
 import tvm
 from tvm.relax.transform import LegalizeOps
-from tvm.script import relax as R, tir as T
+from tvm.script import relax as R, tir as T, ir as I
```

Comment:
Not sure, but I found in other tests we use `@tvm.script.ir_module`

Reply:
The problem is that the printer prints `I.ir_module` by default 

Confirmed with Ruihang that they can substitude each other :)